data-structures/ds-pque/java/PriorityQueue.java

```diff
@@ -0,0 +1,93 @@
+
+import java.util.ArrayList;
+
+public class PriorityQueue {
+    public static class Priorityqueue {
+        ArrayList<Integer> data;
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6336404)